legit_test.py

```diff
@@ -0,0 +1,71 @@
+import legit
+from os.path import expanduser, isdir, exists, join
+from os import makedirs, chmod, walk
+from shutil import rmtree
+
+url = 'https://github.com/katridi/test_folder'
```

Comment:
tests cannot depend on any specific installation config.

Reply:
Yes, you're right.
I need replacement